src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
```

Comment:
Try to use decomposing whenever it is possible 
```suggestion
  for (const { type, extraData, ... } of actions) {
```